### DISCUSSION TITLE
mdn/content now has 2000+ contributors, has ~190+ active contributors a month, gaining ~130+ new contributors a month

### DISCUSSION THREAD

**Discussion by sideshowbarker** [mdn/mdn-community#486] [Content]
The number of contributors who’ve had a PR merged into the mdn/content repo has now exceeded 2000.

To put that number into perspective: As far as I’ve been able to discern, there are in all of GitHub less than 30 repos who have more than 2000 contributors. Here’s the list:

Repo                                                                      | Total contributors  | Uniq contributors per month (average) | New contributors per month (average)
------------------------------------------------------------------------- | ------------: | --------------: | ------------:
[torvalds/linux](https://github.com/torvalds/linux)                       | 24,000+       | ~1100           | ~120
[MicrosoftDocs/azure-docs](https://github.com/MicrosoftDocs/azure-docs)   | 12,000+       | ~850            | ~160
[Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core)       | 11,000+       | ~225            | ~60
[Homebrew/homebrew-cask](https://github.com/Homebrew/homebrew-cask)       | 8,600+        | ~275            | ~70
[ansible/ansible](https://github.com/ansible/ansible)                     | 6,000+        | ~30             | ~10
[rails/rails](https://github.com/rails/rails)                             | 5,400+        | ~65             | ~25
[freeCodeCamp/freeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp) | 4,700+        | ~35             | ~15
[rust-lang/rust](https://github.com/rust-lang/rust)                       | 4,300+        | ~180            | ~40
[gatsbyjs/gatsby](https://github.com/gatsbyjs/gatsby)                     | 4,000+        | ~40             | ~15
[kubernetes/kubernetes](https://github.com/kubernetes/kubernetes)         | 3,900+        | ~120            | ~25
[tensorflow/tensorflow](https://github.com/tensorflow/tensorflow)         | 3,700+        | ~180            | ~20
[nodejs/node](https://github.com/nodejs/node)                             | 3,400+        | ~55             | ~20
[laravel/framework](https://github.com/laravel/framework)                 | 3,300+        | ~85             | ~30
[symfony/symfony](symfony/symfony)                                        | 3,300+        | ~70             | ~25
[facebook/react-native](https://github.com/facebook/react-native)         | 3,000+        | ~55             | ~15
[pytorch/pytorch](https://github.com/pytorch/pytorch)                     | 3,000+        | ~195            | ~45
[home-assistant/core](https://github.com/home-assistant/core)             | 3,000+        | ~165            | ~45
[pandas-dev/pandas](https://github.com/pandas-dev/pandas)                 | 2,800+        | ~55             | ~25
[mui/material-ui](https://github.com/mui/material-ui)                     | 2,600+        | ~50             | ~25
[scikit-learn/scikit-learn](https://github.com/scikit-learn/scikit-learn) | 2,500+        | ~60             | ~30
[django/django](https://github.com/django/django)                         | 2,500+        | ~35             | ~15
[moby/moby](https://github.com/moby/moby)                                 | 2,300+        | ~10             | ~5
[apache/airflow](https://github.com/apache/airflow)                       | 2,300+        | ~90             | ~45
[vercel/next.js](https://github.com/vercel/next.js)                       | 2,100+        | ~80             | ~50
[mdn/content](https://github.com/mdn/content)                             | 2,000+        | ~190            | ~130

But along with having a high number of contributors, there’s the following:

- mdn/content has an exceptionally high number of new contributors coming in each month: over the last 6 months, we have been gaining an average of about 130 new contributors each month
- mdn/content has an exceptionally high number of contributors contributing PRs each month: over the last 6 months, about 190 active contributors each month

The rate at which mdn/content is gaining new contributors is almost unique — In all of GitHub I’ve so far found only one other repo that’s gaining new contributors at a higher rate: the MicrosoftDocs/azure-docs repo.

But the number of people that mdn/content has actively contributing PRs in any given month is also highly exceptional — In all of GitHub, among the set of repos with 2000+ contributors, there are only 10 repos with more than 150 active contributors per month, and only 5 repos with more active monthly contributors than mdn/content: the linux kernel repo, MicrosoftDocs/azure-docs, the homebrew-core and homebrew-cask repos, and the pytorch repo.

**Comment by pepelsbey** [comment]
Closing as outdated/irrelevant.
